### PR TITLE
Fix a typo in NEWS.md

### PR DESCRIPTION
**Pull request by byroot**
FYI @hsbt

### CODE REVIEW

**Comment by hsbt** [comment]
👍  Thanks!

**Review by hsbt — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Note: We're only listing outstanding class updates.
 
 * RubyGems and Bundler warn if users do `require` the following gems without adding them to Gemfile or gemspec.
   This is because they will become the bundled gems in the future version of Ruby. This warning is suppressed
-  if you use bootsnap gem. We recoomend to run your application with `DISABLE_BOOTSNAP=1` environmental variable
+  if you use bootsnap gem. We recomend to run your application with `DISABLE_BOOTSNAP=1` environmental variable
```
Comment (junaruga) [inline NEWS.md:132]: Not "recomend" but "recommend".

**Review by junaruga — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 1 above)
Comment (byroot) [reply NEWS.md:132]: Ahaha, good call

**Review by junaruga — APPROVED** [review]
It looks good to me now!

**Pull request merged**